_posts/2016-12-07-forget-the-keyboard.md

```diff
@@ -49,7 +49,8 @@ When your fingers are on the home row, you know that if you type with *that* fin
 
 ![Position of letter j](static/img/hands-keyboard-j.jpg)
 
-Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. <br/>**Start thinking that your right hand index is the `J` letter**. As long you don't move your hands from the home row, that finger will always be the **`J`**. It's a simple association that your muscle memory will easily remember.
+Well, that's all you need to know  about **`J`**. Do you mind *where* the **`J`** key is, on the keyboard? No, you don't. Just don't think to the physical key under your finger. <br/>
```

Comment:
As stated before, I like to have this sentence emphasized in a new line, but I keep thinking it belongs to the same paragraph. I'm reverting this commit.